vignettes/utMetrics.Rmd

```diff
@@ -0,0 +1,121 @@
+---
+title: "utMetrics"
+output: 
+  prettydoc::html_pretty:
+    theme: architect
+    highlight: vignette
+    #toc: yes
+
+vignette: >
+  %\VignetteIndexEntry{utMetrics}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup, echo=FALSE}
+library(utMetrics)
+```
+
+```{r echo=FALSE, out.width = "55%", fig.align = "center"}
+knitr::include_graphics("metrics.avif")
+```
+
+utMetrics provides the official metrics of Utah Tech University. These metrics can be used in both internal and external presentations regarding Utah Tech University. If you need metrics that need to come from or match USHE or IPEDS, please reach out to the Insitutional Effecitiveness office (datablaze@utahtech.edu) to receive those metrics.
+
+## Metric Names
+
+Metric names are designed to give a clear understanding of the metric.
+
+Metric names are derived from the components of the Metric.
+
+- Season (if applicable)
+- Computation
+- by
+- Partition(s)
```

Comment:
@andreabr-dixie I think the by and the partition are the same idea. Do we want to make this single line?